docs/src/main/sphinx/release/release-479.md

```diff
@@ -0,0 +1,98 @@
+# Release 479 (TBD 2025)
+
+## General
+
+* Support automatic TLS certificate generation in the `ANNOUNCE` node discovery mode. ({issue}`27030`)
+* Fix planning failure of certain queries involving empty VALUES. ({issue}`21506`)
+* Add support for specifying the location of the Exchange Manager configuration file. ({issue}`26611`)
+* Publish query finishing time event listener. ({issue}`27202`)
```

Comment:
@chenjian2664 Could you sort entires by the following rule? 
* New features: Start with Add or Add support for or similar wording
* Performance improvements: Start with Improve or Improve performance or similar wording
* Bug fixes: Start with Fix or Prevent or similar wording

https://trino.io/development/process#release-note-guidelines